joern-cli/frontends/rubysrc2cpg/src/main/antlr4/io/joern/rubysrc2cpg/parser/RubyParser.g4

```diff
@@ -110,16 +110,16 @@ primary
 // --------------------------------------------------------
 
 singleLeftHandSide
-    :   variableIdentifier
-    |   primary LBRACK arguments? RBRACK
-    |   primary (DOT | COLON2) (LOCAL_VARIABLE_IDENTIFIER | CONSTANT_IDENTIFIER)
-    |   COLON2 CONSTANT_IDENTIFIER
+    :   variableIdentifier                                                                                          # variableIdentifierOnly
```

Comment:
W.r.t. to labels, please follow the convention of suffixing the rule (name) they are part of. To wit, lables under `expressionOrCommand` are named `invocationExpressionOrCommand`, `notExpressionOrCommand`, etc. This shall avoid any future name collisions and hopefully be easier to reason about.

Reply:
Agreed and done. Some of the names are really long however. Please let me know if you have any further suggestions. Marking this resolved.